front/src/modules/ui/data/data-table/components/DataTableHeaderPlusButtonContent.tsx

```diff
@@ -0,0 +1,50 @@
+import { useCallback } from 'react';
+
+import { FieldMetadata } from '@/ui/data/field/types/FieldMetadata';
+import { IconPlus } from '@/ui/display/icon';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { MenuItem } from '@/ui/navigation/menu-item/components/MenuItem';
+import { useRecoilScopedValue } from '@/ui/utilities/recoil-scope/hooks/useRecoilScopedValue';
+
+import { useTableColumns } from '../hooks/useTableColumns';
+import { TableRecoilScopeContext } from '../states/recoil-scope-contexts/TableRecoilScopeContext';
+import { hiddenTableColumnsScopedSelector } from '../states/selectors/hiddenTableColumnsScopedSelector';
+import { ColumnDefinition } from '../types/ColumnDefinition';
+
+export const DataTableHeaderPlusButtonContent = () => {
+  const { closeDropdown } = useDropdown();
```

Comment:
as DataTableHeaderPlusButtonContent.tsx is called within DropdownScope, useDropdown is able to retrieve the right dropdown scope id by itself